src/AzureFunctions.PowerShell.OpenTelemetry.SDK/Traces/StartOpenTelemetryInvocationInternal.cs

```diff
@@ -42,6 +43,14 @@ public class StartOpenTelemetryInvocationInternal : PSCmdlet
         protected override void ProcessRecord()
         {
             var newActivity = FunctionsActivityBuilder.StartInternalActivity(InvocationId, TraceParent, TraceState);
+
+            try
+            {
+                string script = "param($id) Set-FunctionInvocationContext -InvocationId $id";
+                _ = this.InvokeCommand.InvokeScript(script, false, PipelineResultTypes.Output, null, new object[] { InvocationId });
+            }
+            catch { }
```

Comment:
Catching all possible exceptions _without rethrowing_ always looks suspicious. What are the problems we anticipate that are safe to ignore here? For example, I'm quite sure we shouldn't handle `OutOfMemoryException` like this. But can we come up with a list of anticipated exception types instead? What happens if we remove this `catch` completely?